library/kotlin/src/io/envoyproxy/envoymobile/Element.kt

```diff
@@ -0,0 +1,16 @@
+package io.envoyproxy.envoymobile
+
+import java.util.regex.Pattern
+
+/**
+ * Element for stats.
+ */
```

Comment:
Can you document what the intended value is supposed to be (given the regex below)?